tools/picard.py

```diff
@@ -152,6 +152,8 @@ def execute(self, inBam, outFastq1, outFastq2, outFastq0=None,
         trim reads at the clipping position specified by the Illumina clipping attribute
         (which is defined by the class variable SamToFastqTool.illumina_clipping_attribute).'''
 
+        assert os.path.isfile(inBam)
+
         if tools.samtools.SamtoolsTool().isEmpty(inBam):
             # Picard SamToFastq cannot deal with an empty input BAM file
```

Comment:
@yesimon says this empty input hardening is no longer necessary, so since it is removed upstream it can be here too.